### ISSUE TITLE
German translation for "page" missing

### ISSUE THREAD

**Issue by meisterluk** [sphinx-doc/sphinx#2779]
Consider `sphinx/writers/latex.py`.
1. Let `language='de'`
2. line 536: `prefix = '\\addto\\captions%s{' % self.babel.get_language()`
3. (1.) + (2.) ⇒ `self.babel.get_language() == 'ngerman'`
4. line 444: `self.elements['pageautorefname'] = self.babel_defmacro('\\pageautorefname', self.encode(_('page')))`
5. (1.) + (4.) ⇒ `self.encode(_('page')) == _('page') == 'page'`
6. (5.) ⇒ `\addto\extrasngerman{\def\pageautorefname{page}}`
7. `latex_show_pagerefs = True`
8. (7.) ⇒ Input reSt: `:ref:`im Kapitel Zusammenfassung <conclusion>``
9. (8.) ⇒ `im Kapitel Zusammenfassung (page 42)`

Simply put: `sphinx/locale/de/LC_MESSAGES` misses translation `Seite` for `page`.
Source for translation: I'm a german native speaker.

**Comment by tk0miya** [comment]
Now we are translate the messages at Transifex.
https://www.transifex.com/sphinx-doc/sphinx-1/

It seems the message is not translated yet. Now Germany messages are translated by 96%.
Please join the team and translate them :-)

Thanks,

**Comment by meisterluk** [comment]
I did. Waiting for confirmation.

**Comment by meisterluk** [comment]
Confirmed. I added the translation. Therefore closing this issue. Thanks @tk0miya!

**Comment by tk0miya** [comment]
Thank you for contribution. Your translation will be included at next release :-)
